helm/aws-ebs-volume-tagger/values.yaml

```diff
@@ -7,6 +7,8 @@ project:
 
 image:
   registry: quay.io
-  name: giantswarm/{APP-NAME}
+  name: giantswarm/aws-ebs-volume-tagger
   tag: v0.0.1
```

Comment:
I think you need to use `[[ .Version ]]` here in order for automation to use the right version based on the release or the SHA of the PR.

Reply:
good point, will do that 👍🏻